pkg/termcorder/termcorder_test.go

```diff
@@ -0,0 +1,116 @@
+package termcorder_test
+
+import (
+	"bytes"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/haguro/termcord/pkg/termcorder"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewTermcording(t *testing.T) {
+	t.Parallel()
+	cfg := &termcorder.Config{}
+	tc, err := termcorder.NewTermcording(cfg)
+	assert.IsType(t, &termcorder.Termcording{}, tc)
+	assert.NoError(t, err)
```

Comment:
There's nothing wrong with using assert-style libraries like `testify`, but I think it's important to be thoroughly familiar with both the standard `testing` library, and what we might call the standard Go testing style. For example, in standard style we would write:

```go
if err != nil {
	t.Errorf("want no error with empty config, but got %v", err)
}
```